backend/src/main/resources/db/migration/V1.23.0__NOTIF-291_events.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE event (
+   id UUID NOT NULL,
+   account_id VARCHAR(50) NOT NULL,
+   event_type_id UUID,
+   payload TEXT,
+   created TIMESTAMP NOT NULL,
+   CONSTRAINT pk_event PRIMARY KEY (id),
+   CONSTRAINT fk_event_event_type_id FOREIGN KEY (event_type_id) REFERENCES event_type (id) ON DELETE CASCADE
+);
+
+ALTER TABLE notification_history
+    DROP COLUMN event_id,
```

Comment:
This `DROP` is safe on prod. I verified the database content and the `event_id` field always contains an empty String.